releasenotes/config.yaml

```diff
@@ -12,6 +12,11 @@ template: |
           #
           # Each section note must be formatted as reStructuredText.
           ---
+          upgrade:
```

Comment:
this is cool, but I guess what actually determines the order of the sections in the generated changelog is what's defined under the `sections` key below no?

Reply:
🤦‍♂️